src/messages/runtime_not_found.cr

```diff
@@ -0,0 +1,9 @@
+message RuntimeFileNotFound do
```

Comment:
Please, rename this file according to the message name, i.e.: `runtime_file_not_found.cr`

Reply:
No worries :)